CONTRIBUTING.md

```diff
@@ -206,7 +206,19 @@ Make sure that the email body contains:
 
 The easiest way to report a WSL process crash is by [collecting a user-mode crash dump](https://learn.microsoft.com/en-us/windows/win32/wer/collecting-user-mode-dumps).
 
-To enable automatic crash dumps, run the following commands in an elevated command prompt:
+To force WSL to crash and collect this information, please open a PowerShell prompt with admin privileges, navigate to a folder where you'd like to put your log files and run these commands: 
```

Comment:
This script won't technically force WSL to crash. Maybe we could rewrite as "To collect dumps of all running WSL processes" ?